lib/_stream_readable.js

```diff
@@ -121,6 +121,9 @@ function Readable(options) {
   // legacy
   this.readable = true;
 
+  if (options && util.isFunction(options.read))
+    this._read = options.read;
+
```

Comment:
Hm, I'm -0 on adding this if it's a shorthand for `_read = fn` – accepting functions into the constructor is a natural jumping off point for the other changes we'd like to see with streams, and if we don't take that opportunity when we add this functionality we'll have lost the chance to do so in the future (this roughly echoes my position from the other issue.)